ChangeLog

```diff
@@ -1,5 +1,15 @@
 2016-05-09  Terry Leitch  <tleitch1@jhu.edu>
 
+<<<<<<< HEAD
```

Comment:
This is merge conflict message line.  Please fix/remove it.

These change were not made on May 9.  Please create a new entry.